owa/app/js/components/manage_patient_identifier_sources/edit_patient_identifier_sources/localPoolIdentifiers.jsx

```diff
@@ -126,10 +126,11 @@ export default class EditLocalPoolIdentifiers extends Component {
             <tr>
               <td className="editPatientIdentifierSourcesTitle">Pool Identifier Source:</td>
               <td>
-                <Input id="sourceUuid" name="sourceUuid" value={this.state.sourceUuid} onChange={this.handleSelectChange} className="editPatientIdentifierSourcesInput" size="sm" type="select">
+                <Input id="sourceUuid" name="sourceUuid" value={this.state.sourceUuid ? this.state.sourceUuid : ""} onChange={this.handleSelectChange} className="editPatientIdentifierSourcesInput" size="sm" type="select">
+                  <option  value=""></option>
                   {
                     this.state.sources.map((options, index) => {
-                      return(<option key={index} value={options.uuid}>{options.display}</option>);
+                      return(<option key={index} value={options.uuid ? options.uuid : ""}>{options.display}</option>);
```

Comment:
Why did you opt to return an empty string,  when `options.uuid` is not found?

Reply:
React does not allow the value of the select field to be null.

Reply:
Would it be possible to return a message maybe telling a user that the value they expected cannot be found rather than returning an empty string?

Reply:
@LWanjiru The empty string is also an option to the user. the user can choose not to have any value in the select this way.

Reply:
@adamsdenniskariuki would you mind sharing a screenshot of before and after, when the `options.uuid` doesn't exist?

Reply:
<img width="793" alt="select" src="https://user-images.githubusercontent.com/25571987/32799999-4541db92-c98a-11e7-880c-b0a653c00f49.png">

Reply:
@dkayiwa Hope I understood correctly. This is what the select looks like. The user can be able to choose one of the options including none.